### ISSUE TITLE
How to run emacsclient with emacs --daemon from this appimage?

### ISSUE THREAD

**Issue by L0cutus** [probonopd/Emacs.AppImage#15]
thanks !

**Comment by probonopd** [comment]
`./Emacs.AppImage --daemon` does not work?

https://github.com/probonopd/Emacs.AppImage/issues/14

**Comment by L0cutus** [comment]
> `./Emacs.AppImage --daemon` does not work?
> 
> #14

yes, but HOW to run then "emacsclient" that is supposed to be included into the appimage?

**Comment by probonopd** [comment]
This is currently not implemented iirc.
